src/main/drivers/barometer/barometer_dps310.h

```diff
@@ -0,0 +1,51 @@
+/*
+ * This file is part of Cleanflight and Betaflight.
+ *
+ * Cleanflight and Betaflight are free software. You can redistribute
+ * this software and/or modify this software under the terms of the
+ * GNU General Public License as published by the Free Software
+ * Foundation, either version 3 of the License, or (at your option)
+ * any later version.
+ *
+ * Cleanflight and Betaflight are distributed in the hope that they
+ * will be useful, but WITHOUT ANY WARRANTY; without even the implied
+ * warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Derived from INAV (See the original INAV copyright notice below).
+ */
+
+/*
+ * This file is part of INAV.
```

Comment:
I'd just change this to `This file is part of INAV, Cleanflight, and Betaflight.` and then omit the header above - this one is a superset of ours, save for some minor differences.